pymilo/pymilo_obj.py

```diff
@@ -99,6 +99,11 @@ def __init__(self, file_adr, json_dump=None):
             else:
                 with open(file_adr, 'r') as fp:
                     serialized_model_obj = json.load(fp)
+            if "pymilo_version" not in serialized_model_obj:
+                print("TODO error handling")
+                return
+            if not serialized_model_obj["pymilo_version"] == PYMILO_VERSION:
+                print("Raise warning")
```

Comment:
I think we should either `pass` and put `#TODO` comments or raise errors, right?